src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -139,24 +141,28 @@ public InstructorAttributes(Instructor instructor) {
         } else {
             this.role = instructor.getRole();
         }
+        
         this.isDisplayedToStudents = instructor.isDisplayedToStudents();
+        
         if (instructor.getDisplayedName() == null) {
             this.displayedName = DEFAULT_DISPLAY_NAME;
         } else {
             this.displayedName = instructor.getDisplayedName();
         }
+        
         if (instructor.getInstructorPrivilegesAsText() == null) {
             this.privileges = new InstructorPrivileges(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER);
             this.instructorPrivilegesAsText = this.getTextFromInstructorPrivileges();
         } else {
             this.instructorPrivilegesAsText = instructor.getInstructorPrivilegesAsText();
         }
+        
         this.privileges = this.getInstructorPrivilegesFromText();
     }
     
     @Deprecated
     public InstructorAttributes() {
-        
+        // To check during review, why not private it
```

Comment:
Ok - I will take note not to do it again. But how should we handle it? There's one method that is similar but takes in paramters and is also deprecated so I assume this particular method is not supposed to be used in anyway, should we privatize it to prevent it from being instantiated?